### DISCUSSION TITLE
TDengine 对单key 查询效果怎么样？

### DISCUSSION THREAD

**Discussion by powersam** [taosdata/TDengine#4705] [General]
hello,every one!
    请教下由于业务需要针对单个传感器进行查询，可以理解为 where key=xxx and time=xxxx.
    请问TDengine 对此类查询的效率如何?

**Comment by sangshuduo** [comment]
TDengine 不是传统关系数据库，只有 timestamp 和超级表的第一个 TAG 有索引
